pkg/kubelet/cm/cpumanager/cpu_manager.go

```diff
@@ -481,11 +462,7 @@ func (m *manager) updateContainerCPUSet(containerID string, cpus cpuset.CPUSet)
 		})
 }
 
-func (m *manager) GetCPUs(podUID, containerName string) []int64 {
-	cpus := m.state.GetCPUSetOrDefault(string(podUID), containerName)
-	result := []int64{}
-	for _, cpu := range cpus.ToSliceNoSort() {
```

Comment:
I see the old implementation of this used `ToSliceNoSort()`, but your updated use on https://github.com/kubernetes/kubernetes/pull/98019/files#diff-c07d343a8d459721656cf7ddacda71506bde01a5cac288d15a4aba6bfc9a1caeR1031 does not use a `NoSort()` variant.

Reply:
nice catch, done